js/getlocation.js

```diff
@@ -13,8 +13,10 @@ function isLocal (address) {
 
 var getLocation = module.exports = function (ipfs, multiaddrs, cb) {
   if (multiaddrs.length === 0) return cb(null, null)
-  var address = multiaddrs[0].split('/')[2]
-  if (isLocal(address)) return getLocation(ipfs, multiaddrs.slice(1), cb)
+  var current = multiaddrs[0].split('/')
+  var address = current[2]
+  // No support for ip6 lookups currently
+  if (isLocal(address) || current[1] === 'ip6') return getLocation(ipfs, multiaddrs.slice(1), cb)
```

Comment:
Nope as far as I know standard only errors on multiline ifs without curlys, which is what I adopted, no curlys if one line, curlys for multiline. Sounds like your vim plugin needs some updates ;)

But I don't have strong feelings about it, I just don't like curlys if I can avoid them ;)